tests/test_plotting.py

```diff
@@ -90,7 +90,7 @@ def test_1d_p(self):
     def test_1d_elongation(self):
         """Test plotting 1d elongation as a function of toroidal angle."""
         eq = get("precise_QA")
-        grid = LinearGrid(N=20, NFP=eq.NFP)
+        grid = LinearGrid(M=eq.M_grid, N=20, NFP=eq.NFP)
```

Comment:
Until #1138 is resolved, users will not be warned that poloidal resolution is required to plot this, giving junk like
![result](https://github.com/user-attachments/assets/15a1c500-4ced-4153-87b3-6f7d56ffb594)

Reply:
This gave correct plot prior to this PR because the `resolution_requirement=rt` instead of  `resolution_requirement=t` and `eq.compute` would automatically construct a lineargrid with radial and poloidal resolution to compute and then copy data back to 1d grid to plot. Now that it's just  `resolution_requirement=t` we don't make a new grid automatically. I don't think we need to add logic to that automatically since `eq.compute` will raise a warning .

Reply:
I would prefer to add logic to `eq.compute` for that case so that it "just works", especially for plotting stuff. It especially feels weird if you need to create a 2d grid to make a 1d plot.

Reply:
In this PR now, will `plot_1d("a_major/a_minor")` work by default (i.e. will the default grid be sufficient)? If this is a quantity that needs the poloidal resolution, I think the .compute logic should by default compute it on the right grids

Reply:
There is not any logic in `master` to automatically override to a grid with sufficient resolution when the resolution requirement is a single coordinate. We have only done this for volume quantities and surface quantities.

This PR corrects the resolution requirement string of dependencies of `a_major/a_minor` to be only poloidal resolution rather than poloidal and toroidal. So, we are now in an uncovered case where the grid overriding doesn't happen automatically. Instead, a warning is raised to user to increase XX resolution. 

I don't think it's worth adding the three more cases to `eq.compute` to cover this. One could just use one of the existing cases, computing a quantity that only needs poloidal resolution on a flux surface grid. However, perhaps the user's quantity requires significant memory and they can't afford computing it on that higher resolution grid when it doesn't change the output.

Feel free to make the change if you prefer.